x-pack/plugins/cases/common/api/runtime_types.test.ts

```diff
@@ -101,15 +100,5 @@ describe('runtime_types', () => {
 
       expect(decodeWithExcessOrThrow(schemaRt)({ a: 'hi' })).toStrictEqual({ a: 'hi' });
     });
-
-    describe('BulkCreateCommentRequestRt', () => {
```

Comment:
Assuming `BulkCreateCommentRequestRt` is now `BulkCreateAttachmentsRequestRt`, I could not find this test anymore.

Was this replaced by the ones in `x-pack/plugins/cases/common/types/api/attachment/v1.test.ts`? Those do `BulkCreateAttachmentsRequestRt.decode` 😬

Reply:
The logic of this specific test is the same as in `x-pack/plugins/cases/common/types/api/attachment/v1.test.ts`. It does not matter if we use `decode` or `decodeWithExcessOrThrow` (only for this test).